hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -111,13 +119,18 @@ set_user_cntl_synchronizer -name opentitan_2ff $prim_2ff_modules
 # There are two such paths: One path is a the error pin directly into ibex, and the other is ibex's internal check
 set async_fifo_data [get_pins -of_objects [get_cells -hier * -filter {ref_name == prim_fifo_async}] -filter {name =~ rdata_o*}]
 set_ignore_cdc_paths -name async_fifo_to_ibex_data_err -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/data_err_i]
-set_ignore_cdc_paths -name async_fifo_to_ibex_ecc_err -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/u_ibex_core/load_store_unit_i/load_intg_err_o]
+set_ignore_cdc_paths -name async_fifo_to_ibex_ecc_err -through_signal $async_fifo_data -through_signal [get_pins top_earlgrey/u_rv_core_ibex/u_core/u_ibex_core/load_store_unit_i/load_resp_intg_err_o]
```

Comment:
gasp! did this solve a bunch of problems?!

Reply:
Not too many.. hehe